### PR TITLE
Please check the exc. 1

### PR DESCRIPTION
**Pull request by Amit833**

### CODE REVIEW

File: amit/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+let apples = 20;
+let oranges = 30;
+let equal = apples == oranges;
+console.log(equal); //false
+
+let equal2 = apples != oranges;
```
Comment (ciamac-da) [inline amit/index.js]: Better to use !== instead of !=

File: amit/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+console.log(check);
+
+//que.7
+let sub = apples - mangoes;
+let div = oranges / mangoes;
+let check2 = sub < div;
+console.log(check2);
+
+//que 8
+let allEqual = (mangoes == apples) == oranges;
```
Comment (ciamac-da) [inline amit/index.js:36]: let allEqual = (mangoes == apples== oranges);

File: amit/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+console.log(allEqual);
+
+// que 9. Check whether the remainder of apples divided by mangoes and the remainder of oranges divided by mangoes are equal.
```
Comment (ciamac-da) [inline amit/index.js:39]: The solution of 9th exercise would be like this:
console.log((apples % mangoes) == (oranges % mangoes));

File: amit/index.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+let result = div2 == div3;
+console.log(result);
+
+// que 10 Check whether mangoes added to apples is greater than oranges minus mangoes. If it is not, find an operator which will give a result of true.
+
+let plus = mangoes + apples;
+let minus = oranges - mangoes;
+let res1 = plus > minus;
+console.log(res1);
+
```
Comment (ciamac-da) [inline amit/index.js:61]: Get rid of line 58 and 59 and add  console.log( plus >= minus ); instead!

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hey Amit! Please read my comments and try to fix the code!

File: amit/index.js (hunk 2 above)
Comment (Amit833) [reply amit/index.js:36]: but prettier do this for me  let allEqual = (mangoes == apples) == oranges;

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**